lib/components/viewers/stop-viewer.js

```diff
@@ -116,6 +116,10 @@ class StopViewer extends Component {
     this.setState({ date })
   }
 
+  _zoomToStop = () => {
+    this.props.zoomToStop(this.props.stopData)
+  }
```

Comment:
This `zoomToStop` prop could have be a pure action that handles the retrieval of the `stopData` from the redux state all within the action.

Reply:
What if we wanted to allow for zooming to "non-focused" stops? I'd prefer to make this accept an arg.